example-auth-simple.js

```diff
@@ -0,0 +1,183 @@
+  }
+
+  // AddRoots parameters
+  const ADD_ROOTS_PARAMS = {
+    clientDataSetId: 12345,
+    firstRootId: 1,
+    rootDataArray: [
+      {
+        cid: 'baga6ea4seaqpy7usqklokfx2vxuynmupslkeutzexe2uqurdg5vhtebhxqmpqmy',
+        rawSize: 1024
+      },
+      {
+        cid: 'baga6ea4seaqkt24j5gbf2ye2wual5gn7a5yl2tqb52v2sk4nvur4bdy7lg76cdy',
+        rawSize: 2048
+      },
+      {
+        cid: 'baga6ea4seaqjtovkwk4myyzj56eztkh5pzsk5upksan6f5outesy62bsvl4dsha',
+        rawSize: 4096
+      }
+    ]
+  }
+
+  // ScheduleRemovals parameters
+  const SCHEDULE_REMOVALS_PARAMS = {
+    clientDataSetId: 12345,
+    rootIds: [1, 3, 5, 7]
+  }
+
+  // DeleteProofSet parameters
+  const DELETE_PROOF_SET_PARAMS = {
+    clientDataSetId: 12345
+  }
+
+  // ============================================================
+  // SIGNATURE GENERATION - No need to edit below this line
+  // ============================================================
+
+  try {
+    console.log('Synapse Auth Signature Generator')
+    console.log('================================\n')
+
+    // The PDP service contract address is now hardcoded in the SDK constants
+    // but we show it here for reference
+    console.log(`PDP Service Contract: ${PDP_SERVICE_CONTRACT_ADDRESS}`)
+    console.log('(SimplePDPServiceWithPayments on calibration testnet)\n')
+
+    // Initialize Synapse
+    const synapse = await Synapse.create({
+      privateKey: PRIVATE_KEY,
+      rpcURL: RPC_URL
+    })
+
+    const signerAddress = await synapse.getSignerAddress()
+    console.log(`Signer Address: ${signerAddress}`)
+    console.log(`Network: ${RPC_URL}\n`)
+
+    console.log('Generated Signatures:')
+    console.log('===================\n')
+
+    // 1. CreateProofSet
+    console.log('1. CreateProofSet')
+    console.log(`   Client Dataset ID: ${CREATE_PROOF_SET_PARAMS.clientDataSetId}`)
+    console.log(`   Payee: ${CREATE_PROOF_SET_PARAMS.payee}`)
+
+    const createSig = await synapse.signCreateProofSet(
+      CREATE_PROOF_SET_PARAMS.clientDataSetId,
+      CREATE_PROOF_SET_PARAMS.payee
+    )
+
+    console.log(`   Signature (hex): ${createSig.signature}`)
+    console.log(`   Auth Blob (hex): 0x${Buffer.from(createSig.signature.slice(2), 'hex').toString('hex')}`)
+    console.log(`   Encoded Data: ${createSig.signedData}`)
+    console.log()
+
+    // 2. AddRoots
+    console.log('2. AddRoots')
+    console.log(`   Client Dataset ID: ${ADD_ROOTS_PARAMS.clientDataSetId}`)
+    console.log(`   First Root ID: ${ADD_ROOTS_PARAMS.firstRootId}`)
+    console.log(`   Roots to Add: ${ADD_ROOTS_PARAMS.rootDataArray.length}`)
+    ADD_ROOTS_PARAMS.rootDataArray.forEach((root, i) => {
+      console.log(`     [${i}] CID: ${root.cid.toString().substring(0, 20)}... (size: ${root.rawSize})`)
+    })
+
+    const addRootsSig = await synapse.signAddRoots(
+      ADD_ROOTS_PARAMS.clientDataSetId,
+      ADD_ROOTS_PARAMS.firstRootId,
```

Comment:
The biggest thing we have to sync on for threading all the data together is that firstRootId is going to require going to PDPVerifier (proxy) getter